### PR TITLE
[ES] Update `osu!supporter`

### PR DESCRIPTION
**Pull request by ImMock**
Updated to the english version

Waiting for review

### CODE REVIEW

File: wiki/osu!supporter/es.md [hunk 1]
```diff
@@ -1,63 +1,151 @@
...
+
+osu! ofrece a los supporters límites incrementados en varias funciones en línea:
+
+| Valor | Limite regular | Limite de osu!supporter |
+| :-- | :-: | :-: |
+| [Beatmaps pendientes](/wiki/Beatmap/Category#work-in-progress-and-pending) | `4 + min(beatmaps rankeados, 2)` | `8 + min(beatmaps rankeados, 4)` |
+| Beatmaps favoritos | 100 | 1000 |
+| Amigos | 250 | 500 |
+
+Adicionalmente, los usuarios con osu!supporter tienen límites de descarga de beatmaps poco estrictos.
```
Comment (Repflez) [inline wiki/osu!supporter/es.md:91]:
```diff
-poco estrictos.
+menos estrictos.
```

**Review by Repflez — CHANGES_REQUESTED** [review]

File: wiki/osu!supporter/es.md (hunk 1 above)
Comment (ImMock) [reply wiki/osu!supporter/es.md]: All done

**Review by Repflez — APPROVED** [review]

File: wiki/osu!supporter/es.md [hunk 2]
```diff
@@ -1,63 +1,151 @@
...
+*Nota: Cambios menores de nombre podrán ser solicitados al equipo de apoyo de cuentas de manera gratuita-véase [Centro de Ayuda/Cuenta § Puedo cambiar mi nombre de usuario?](/wiki/Help_centre/Account#name-changes).*
+
+El obtener osu!supporter viene con la habilidad de cambiar tu nombre de usuario por primera vez de manera gratuita, siguiendo las [condiciones usuales](/wiki/Help_centre/Account#name-changes).
+
+### osu!direct
+
+osu!direct es el motor de búsqueda y descarga de mapas disponible para supporters.  Podrás acceder desde el menú principal al hacer clic en la pestaña de `osu!direct` en la parte derecha de la pantalla. Además, contener un listado de beatmaps incorporado, osu!direct ofrece otras maneras de obtener nuevos beatmaps sin salir del juego:
+
+- Enlaces de beatmaps en el chat, que te dirigen al sitio web de osu!, ahora activan el aviso de descarga dentro del juego
+- Descargas automáticas de beatmaps mientras juegas en una sala [multijugador](/wiki/Gameplay/Multiplayer) o te encuentras [espectando](/wiki/Gameplay/Spectating) a otro usuario, como una [configuración opcional](/wiki/Client/Options#online)
```
Comment (Walavouchey) [inline wiki/osu!supporter/es.md:71]: missed one bullet point here

**Review by Walavouchey — DISMISSED** [review]

File: wiki/osu!supporter/es.md (hunk 2 above)
Comment (ImMock) [reply wiki/osu!supporter/es.md:71]: Added the missing bullet point

**Review by Repflez — APPROVED** [review]

**Pull request merged**